tox.ini

```diff
@@ -1,21 +1,24 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-{flake8,pylint,unit}
+    py27-{yamllint,ansible_syntax,generate_validation}
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
+skip_install=True
 deps =
+    -rrequirements.txt
```

Comment:
Previously the packages in requirements.txt were pulled in through the install process, which was not allowing us to take advantage of cached packages.